File: src/fib.java
public class fib {
    public static int thefibber(int number) {
        if (number < 2)
            return number;
        else
            return thefibber(number - 1) + thefibber(number - 2);
    }
    public static void main(String[] args) {
        int test_num = 35;
        int actual = 0;
        long start, stop; // clocks
        double avg_time;
        long total_time, run_time, counter;
        total_time = 0;
        for (counter = 0; counter < 10; counter++) {
            start = System.nanoTime();
            actual = thefibber(test_num);
            stop = System.nanoTime();
            run_time = stop - start;
            System.out.println(run_time + " nanoseconds");
            total_time = total_time + run_time;
        }
        avg_time = total_time / counter;
        System.out.println("Lang:Java,Result:" + actual + ",Runtime:" + avg_time/(double)1E9);
    }
}

File: src/fib.c
#include <stdio.h>
#include <time.h>

int fib(int num) {
    return num < 2 ? num : fib(num-1) + fib(num-2);
}

main () {
    int test_num = 35;
    int actual;
    clock_t start, stop;
    double avg_time, total_time, counter, time_in_clock_ticks, time_in_seconds;
    for (counter = 0; counter < 10; counter++) {
        start = clock();
        actual = fib(test_num);
        stop = clock();
        time_in_clock_ticks = stop-start;
        time_in_seconds = (double)time_in_clock_ticks / (double)CLOCKS_PER_SEC;
        // printf("%f\n", time_in_seconds);
        total_time = total_time + time_in_seconds;
    }
    avg_time = total_time / (counter);
    printf("Lang:C(Clang-3.5),Result:%d,Runtime:%f\n", actual, avg_time);
}
